server/GlideAction.ts

```diff
@@ -0,0 +1,11 @@
+import { GlideRecord } from './GlideRecord';
+declare class GlideAction {
+  /**
+   * Set a redirect URL in UI Actions.
+   * @param target takes a GlideRecord object or string query
+   * Examples from official ServiceNow documentation can be seen on 'Create a UI routing action' tutorial
+   */
+  setRedirectURL(target: GlideRecord | string): void;
```

Comment:
```suggestion
  setRedirectURL(target: GlideRecord<any> | string): void;
```
Since GlideRecord is generic and defaults to a `never` GlideRecord, `any` is a good idea so it can receive any GlideRecord